### ISSUE TITLE
Generated temporary SSH key looked up on wrong host

### ISSUE THREAD

**Issue by manzsolutions-lpr** [oVirt/ovirt-ansible-image-template#71]
After successfully generating a template without it I wanted to use `template_prerequisites_tasks` but unfortunately ran into some unexpected behaviour. Apparently Ansible `lookup('file', …)` looks on the machine Ansible is being run from (my machine), not the host it supposedly is executing the task on (the ovirt-engine VM).

When creating the expected file on my local machine (`echo "foo" > /tmp/.image_template.key.pub`) the task would continue successfully albeit it failed at a later stage, presumably due to "foo" not being the correct value. Please note that the key pair is generated on the expected host though.

Code:
https://github.com/oVirt/ovirt-ansible-image-template/blob/master/tasks/qcow2_image.yml#L135

Error message:

```
TASK [ovirt.image-template : Create vm] ******************************************************************************************************************************************************************************************************
[WARNING]: Unable to find '/tmp/.image_template.key.pub' in expected paths (use -vvvvv to see paths)
fatal: [<ovirt-engine host>]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: /tmp/.image_template.key.pub"}
```

**Comment by mnecas** [comment]
Hi @manzsolutions-lpreis,
thank you for the issue please try my patch and tell me if it helped you.

**Comment by manzsolutions-lpr** [comment]
@mnecas I’m terribly sorry I did not respond last week but I’m currently swamped and chose a different approach to achieve my goal after filing the issue (ovirt-ansible-vm-infra + `ovirt_template`) as I would also have to find a work-around for setting up the network to actually get a reachable machine. My hope is that I will get around to test it next week.

**Comment by mnecas** [comment]
Found one issue with #72 which should be resolved in #74.
